packages/commonwealth/client/scripts/views/components/comments/comment.tsx

```diff
@@ -5,8 +5,9 @@ import type { Account, Comment as CommentType } from 'models';
 import moment from 'moment';
 
 import app from 'state';
-import { ContentType } from 'types';
+import { ContentType, ThreadActionType } from 'types';
 import { ChainType } from '../../../../../../common-common/src/types';
+import deleteComment from '../../../../../server/routes/deleteComment';
```

Comment:
We should not be importing routes from the server onto the client.

Also can we make the common-common import just `'common-common/src/types'`?

Reply:
My autoimport pulled it in by accident when I was typing

Reply:
Can we fix the other import while we're in this file?